molgenis-scripts-core/src/main/java/org/molgenis/script/core/exception/TemplateCreationException.java

```diff
@@ -0,0 +1,40 @@
+package org.molgenis.script.core.exception;
+
+import java.io.IOException;
+
+import static java.lang.String.format;
+import static java.util.Objects.requireNonNull;
+
+/**
+ * Wraps exceptions that can occur during the creation of a Template for a script.
+ */
+// S2166 'Classes named like "Exception" should extend "Exception" or a subclass' often gives false positives at dev time
+@SuppressWarnings({ "squid:MaximumInheritanceDepth", "squid:S2166" })
+public class TemplateCreationException extends ScriptGenerationException
```

Comment:
this one is perhaps a good one to make more generic later if we have other ftl render exceptions

Reply:
I'll add it to the TODO doc